core/src/main/resources/spline.default.properties

```diff
@@ -59,6 +59,8 @@ spline.lineageDispatcher.http.timeout.read=120000
 # Should the payload be compressed.
 # If unspecified, the request compression is negotiated with the endpoint, otherwise it's disabled.
 # spline.lineageDispatcher.http.requestCompression=true | false
+# Additional http headers to be sent by dispatcher
+# spline.lineageDispatcher.http.header.X-CUSTOM_HEADER=custom-header-value
 
```

Comment:
Let's add a preconfigured "azur-function" dispatcher with the `X-FUNCTIONS-KEY` set to `USER_API_KEY` with the reference to the following doc page in the comments:

https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=in-process%2Cfunctionsv2&pivots=programming-language-csharp#api-key-authorization

Reply:
Also describe the usage clearly in the README. I believe it will be helpful to many users.